modules/nginx/manifests/redirect.pp

```diff
@@ -0,0 +1,42 @@
+define nginx::redirect (
+    $ssl_crt_source     = "puppet:///modules/localconfig/ssl/*.${web_domain_redirect_from}/server.crt",
```

Comment:
I think you may have removed the `$web_domain` and `$web_domain_redirect_from` here by accident